contrib/steghideexport.lua

```diff
@@ -0,0 +1,323 @@
+    is_directory = true             -- True if the file chooser button only allows directories to be selecte
+}
+
+
+-- Text file dialog
+local textfile_chooser_button = dt.new_widget("file_chooser_button")
+{
+    title = _('Text file'),  -- The title of the window when choosing a file
+    value = ".config/darktable/steghide/steghide_default",
+    reset_callback = function(self) 
+       self.value = ".config/darktable/steghide/steghide_default" 
+    end
+}
+
+
+local selection = dt.gui.selection()
+  local result = ""
+  local array = {}
+  for _,img in pairs(selection) do
+    array[img.path] = true
+  end
+  for path in pairs(array) do
+    if result == "" then
+      result = path
+    else
+      result = result.."\n"..path
+    end
+  end
+  file_chooser_button.value = result
+
+local enc_combobox = dt.new_widget("combobox")
+{
+    label = "", 
+    value = 15, "cast-128 cbc", "cast-128 cfb", "cast-128 ctr","cast-128 ecb","cast-128 ncfb","cast-128 nofb","cast-128 ofb","gost cbc","gost cfb","gost ctr","gost ecb","gost ncfb","gost nofb","gost ofb","rijndael-128 cbc","rijndael-128 cfb","rijndael-128 ctr","rijndael-128 ecb","rijndael-128 ncfb","rijndael-128 nofb","rijndael-128 ofb","twofish cbc","twofish cfb","twofish ctr","twofish ecb","twofish ncfb","twofish nofb","twofish ofb","arcfour stream","cast-256 cbc","cast-256 cfb","cast-256 ctr","cast-256 ecb","cast-256 ncfb","cast-256 nofb","cast-256 ofb","loki97 cbc","loki97 cfb","loki97 ctr","loki97 ecb","loki97 ncfb","loki97 nofb","loki97 ofb","rijndael-192 cbc","rijndael-192 cfb","rijndael-192 ctr","rijndael-192 ecb","rijndael-192 ncfb","rijndael-192 nofb","rijndael-192 ofb","saferplus cbc","saferplus cfb","saferplus ctr","saferplus ecb","saferplus ncfb","saferplus nofb","saferplus ofb","wake stream","des cbc","des cfb","des ctr","des ecb","des ncfb","des nofb","des ofb","rijndael-256 cbc","rijndael-256 cfb","rijndael-256 ctr","rijndael-256 ecb","rijndael-256 ncfb","rijndael-256 nofb","rijndael-256 ofb","serpent cbc","serpent cfb","serpent ctr","serpent ecb","serpent ncfb","serpent nofb","serpent ofb","xtea cbc","xtea cfb","xtea ctr","xtea ecb","xtea ncfb","xtea nofb","xtea ofb","blowfish cbc","blowfish cfb","blowfish ctr","blowfish ecb","blowfish ncfb","blowfish nofb","blowfish ofb","enigma stream","rc2 cbc","rc2 cfb","rc2 ctr","rc2 ecb","rc2 ncfb","rc2 nofb","rc2 ofb","tripledes cbc","tripledes cfb","tripledes ctr","tripledes ecb","tripledes ncfb","tripledes nofb","tripledes ofb",
+    reset_callback = function(self) 
+       self.value = 15 
+    end
+}  
+  
+  
+local widget = dt.new_widget("box") {
+    orientation = "vertical",
+    check_button_selected_file,
+    check_button_usertext,
+    label_textfile,
+    textfile_chooser_button,
+    label_usertext,
+    entrytext,
+    label_password,
+    entrypassword,
+    label_enc,
+    enc_combobox,
+    label_path,
+    file_chooser_button,    
+}
+
+
+
+local function checkIfBinExists(bin)
```

Comment:
@klemmh 
I haven't heard anything from you. What is the status of this scripts?